app/models/concerns/orchestration/dns.rb

```diff
@@ -55,7 +55,11 @@ def rebuild_dns
   end
 
   def queue_dns
-    return unless (dns? || dns6? || reverse_dns? || reverse_dns6?) && errors.empty?
+    return unless dns? || dns6? || reverse_dns? || reverse_dns6?
+    unless orchestration_errors?
```

Comment:
`if orchestration_errors?` ?
If there are errors, we want to log and not queue that.